easybuild/easyconfigs/f/foss_mv2/foss_mv2-2016a.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'Toolchain'
+
+name = 'foss_mv2'
```

Comment:
you're reinventing the existing `gmvolf` toolchain :)

the `foss` toolchain deliberately is not composed of the first letter of every component (like `goolf`, `gmvolf`, ...), to retain the freedom of changing some components at some point in time (e.g. change `OpenMPI` to `MVAPICH2` in future release).

So, maybe stick to `gmvolf` as naming?